### PR TITLE
Mimir 816 react app dashboard

### PR DESCRIPTION
**Pull request by Garlov**

### CODE REVIEW

File: src/main/resources/react4xp/_entries/Dashboard/utils/websocket/WebsocketProvider.jsx [hunk 1]
```diff
@@ -0,0 +1,89 @@
+import React, { createContext } from 'react'
+import PropTypes from 'prop-types'
+import { actions as commonActions } from '../../containers/HomePage/slice'
+import setupStatRegListeners from '../../containers/StatRegDashboard/listeners'
+import setupHomePageListeners from '../../containers/HomePage/listeners'
+import setupDataQuieriesListeners from '../../containers/DataQueries/listeners'
```
Comment (johnnadeluy) [inline src/main/resources/react4xp/_entries/Dashboard/utils/websocket/WebsocketProvider.jsx:6]: setupDataQuieresListeners -> setupDataQueriesListeners?  @ both import and function call

**Review by johnnadeluy — CHANGES_REQUESTED** [review]
Aside from the other known w3c validator errors that can be ignored, there's also: (Error) '_Bad value https://fonts.googleapis.com/css?family=Open+Sans|Open+Sans:700|Roboto:700|Roboto+Condensed:700 for attribute href on element link: Illegal character in query: | is not allowed._' 
And (Warning) '_The type attribute is unnecessary for JavaScript resources._'


Only nitpicky stuff on this review, but the code is very clean and well-structured - easy to follow through! Great job!! 😊

File: src/main/resources/admin/tools/dashboard/dashboard.html [hunk 2]
```diff
@@ -4,7 +4,7 @@
     <meta charset="UTF-8"></meta>
     <meta http-equiv="Cache-control" content="NO-CACHE"></meta>
```
[the remark this answers is not in the document]
Comment (Garlov) [reply src/main/resources/admin/tools/dashboard/dashboard.html:5]: Wont touch this now, we'll have to look at this later

**Review by johnnadeluy — APPROVED** [review]
👏😊

**Pull request merged**